### ISSUE TITLE
 --qase-testops-run-complete not working

### ISSUE THREAD

**Issue by ry-sny** [qase-tms/qase-python#160]
When using ` --qase-testops-run-complete=True` the run is not mark as completed, this only works if I set the environment variable `QASE_TESTOPS_RUN_COMPLETE`

This was reported in https://github.com/qase-tms/qase-python/issues/107 but closed

Is it possible to get a fix for this?

**Comment by NickVolynkin** [comment]
@ry-sny thanks for the issue, we'll look into it.

**Comment by NickVolynkin** [comment]
From `pytest --help` with `qase-pytest` enabled:

```log
  qase_testops_run_title (string):
                        default value for --qase-testops-run-title
  qase_testops_run_description (bool):
                        default value for --qase-testops-run-complete
  qase_testops_run_description (bool):
                        default value for --qase-testops-run-complete
  qase_testops_bulk (bool):
                        default value for --qase-testops-bulk
```

Note the obvious error where `qase_testops_run_description` is mixed with `--qase-testops-run-complete`

**Comment by NickVolynkin** [comment]
We had to rollback the release fixing this issue, so I'm reopening it for a while.

**Comment by NickVolynkin** [comment]
Resolved in `qase-pytest==5.0.5`, but works only with lowercase or numeric values:

```
pytest --qase-testops-run-complete=true
pytest --qase-testops-run-complete=1
```

There's a bug in the configuration parser, parsing `=True` as falsy value. We we will fix in a later release, see issue #176.

**Comment by NickVolynkin** [comment]
@ry-sny we have resolved it in `qase-pytest` version 6.0.1.
